### DISCUSSION TITLE
Welcome to EMHASS Discussions!

### DISCUSSION THREAD

**Discussion by davidusb-geek** [davidusb-geek/emhass#264] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joax01** [comment]
I suggest  EMHASS adhere to the guidelines in the EU Energy Efficiency Directive. The directive is found at this link: 
https://energy.ec.europa.eu/news/new-energy-efficiency-directive-published-2023-09-20_en
Several of the guidelines in the directive are already fulfilled by existing programming means already available in the EMHASS / Home assistant. However, it is my belief that a more detailed analysis would result in additional  parameter settings and graphical visualization methods to assist users to better understand the ideas of modern energy efficiency and facilitate adaptations to national operators specific tariffs and unique settings necessary to handle the various differences in tariffs used in different countries and operators across the EU. The user need means to easily understand and adapt their installations to specific details out of how operators in the EU community plan to implement the directive. As an example, the methods to calculate the average maximum consumption as the basis for calculating the new energy cost vary in the tariff models currently in development:
•	Demand tariffs
•	Peak load pricing
•	Time-of-use (TOU) tariffs
 
All these models share the principles behind the core concept of how to optimize electricity usage with demand tariffs is about avoiding high and sudden power peaks.

Here's a closer explanation:

Why avoid power peaks?

Higher fees:
Demand tariffs reward even electricity usage. High power peaks lead to higher fees.
Load on the electricity grid:
Sudden and high power peaks put a strain on the electricity grid, which can lead to stability problems.
How to regulate electric heating:

Gradual heating:
Instead of turning on the electric heating to full power, start with a lower power and increase it gradually.
This reduces the inrush current spike and evens out the power consumption.
Smart thermostat:
Use a smart thermostat that can control the electric heating based on both temperature and electricity price.
The thermostat can lower the temperature slightly during hours with higher electricity prices and raise it during hours with lower electricity prices.
Zoning:
If possible, divide your home into zones and heat them at different times.
This reduces the total power peak.
Time control:
Use timers to control when the electric heating turns on and off.
This way, you can ensure that the electric heating does not turn on at the same time as other power-intensive appliances.
Example:

Instead of turning on the electric heating with 10 kW for a short time, you can turn it on with 3 kW for a longer time.
Or, if you know you need 10kW, then you can try to turn on 2.5kW at four different times, spread over a longer period of time.
The control of heating systems can become complex to manage concepts like zoning and controls to  spread over a longer period of time.
Important:
- Keep track of your electricity usage using your smart meter or an app.
- Adapt your electricity usage to your lifestyle and needs.
- By being aware of your power peaks and taking active measures to reduce them, you can save money and contribute to a more stable electricity grid.

**Reply by anselmhuewe** [reply]
Wouldn't almost all of these issues be addressed if the users optimizes electricity costs (thus, minimizes them?). Minimizing costs automatically maximizes the use of renewable energies (as they are marginally cheap) and avoides hours of peak power (as they are expensive).
In terms of gradual heating: That's also an implizit outcome of cost minimization, at least for heat pumps, because heat pumps are most efficient when being run as steadily as economically feasible.
Thus, at www.shyft-power.com, I optimize for costs, and haven't seen the need to add alternative optimization targets.
What do you think?
